bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -94,6 +96,12 @@ String getMetadata(CensusInstance censusInstance, String title, String body, Lis
                                         .flatMap(comment -> comment.body().lines())
                                         .filter(line -> METADATA_COMMENTS_PATTERN.matcher(line).find())
                                         .collect(Collectors.joining());
+            commentString = commentString + comments.stream()
+                    .filter(comment -> comment.author().username().equals("mlbridge[bot]"))
```

Comment:
We also could compare `comment.author().id()` here, but I am not sure if it is a good idea to explicitly expose the id of mlbridge bot.

Reply:
The `commentString` looks like it's only containing comments from the currentUser (which is the bot user for the pr bot). I think we need to filter all comments again for the mlbridge user to find this comment.

Reply:
Yes, commentString only containing comments from the currentUser. So I actually filter all comments again in line 99.

Reply:
Oh right, I misread that.